res/controllers/Pioneer-DDJ-SX-scripts.js

```diff
@@ -0,0 +1,2149 @@
+    engine.setValue(
+        group,
+        "filterHigh",
+        script.absoluteNonLin(fullValue, 0.0, 1.0, 4.0, 0, 0x3FFF)
+    );
+};
+
+PioneerDDJSX.filterMidKnobMSB = function(channel, control, value, status, group) {
+    PioneerDDJSX.highResMSB[group].filterMid = value;
+};
+
+PioneerDDJSX.filterMidKnobLSB = function(channel, control, value, status, group) {
+    var fullValue = (PioneerDDJSX.highResMSB[group].filterMid << 7) + value;
+    engine.setValue(
+        group,
+        "filterMid",
+        script.absoluteNonLin(fullValue, 0.0, 1.0, 4.0, 0, 0x3FFF));
+};
+
+PioneerDDJSX.filterLowKnobMSB = function(channel, control, value, status, group) {
+    PioneerDDJSX.highResMSB[group].filterLow = value;
+};
+
+PioneerDDJSX.filterLowKnobLSB = function(channel, control, value, status, group) {
+    var fullValue = (PioneerDDJSX.highResMSB[group].filterLow << 7) + value;
+    engine.setValue(
+        group,
+        "filterLow",
+        script.absoluteNonLin(fullValue, 0.0, 1.0, 4.0, 0, 0x3FFF)
+    );
+};
+
+PioneerDDJSX.deckFaderMSB = function(channel, control, value, status, group) {
+    PioneerDDJSX.highResMSB[group].deckFader = value;
+};
+
+PioneerDDJSX.deckFaderLSB = function(channel, control, value, status, group) {
+    var fullValue = (PioneerDDJSX.highResMSB[group].deckFader << 7) + value;
+
+    if (PioneerDDJSX.shiftPressed &&
+        engine.getValue(group, "volume") === 0 &&
+        fullValue !== 0 &&
+        engine.getValue(group, "play") === 0
+    ) {
+        PioneerDDJSX.chFaderStart[channel] = engine.getValue(group, "playposition");
+        engine.setValue(group, "play", 1);
+    } else if (
+        PioneerDDJSX.shiftPressed &&
+        engine.getValue(group, "volume") !== 0 &&
+        fullValue === 0 &&
+        engine.getValue(group, "play") === 1 &&
+        PioneerDDJSX.chFaderStart[channel] !== null
+    ) {
+        engine.setValue(group, "play", 0);
+        engine.setValue(group, "playposition", PioneerDDJSX.chFaderStart[channel]);
+        PioneerDDJSX.chFaderStart[channel] = null;
+    }
+    engine.setValue(group, "volume", fullValue / 0x3FFF);
+};
+
+PioneerDDJSX.filterKnobMSB = function(channel, control, value, status, group) {
+    PioneerDDJSX.highResMSB[group].filterKnob = value;
+};
+
+PioneerDDJSX.filterKnobLSB = function(channel, control, value, status, group) {
+    var fullValue = (PioneerDDJSX.highResMSB[group].filterKnob << 7) + value;
+    engine.setValue("[QuickEffectRack1_" + group + "]", "super1", fullValue / 0x3FFF);
+};
+
+PioneerDDJSX.crossfaderCurveKnobMSB = function(channel, control, value, status, group) {
+    PioneerDDJSX.highResMSB[group].crossfaderCurveKnob = value;
+};
+
+PioneerDDJSX.crossfaderCurveKnobLSB = function(channel, control, value, status, group) {
+    var fullValue = (PioneerDDJSX.highResMSB[group].crossfaderCurveKnob << 7) + value;
+    script.crossfaderCurve(fullValue, 0x00, 0x3FFF);
```

Comment:
After checking this out, I can say that the crossfader adjustment knob doesn't change the MIDI signals, so the crossfader curve adjustment is completely up the Mixxx.